kloppy/domain/models/common.py

```diff
@@ -119,6 +119,25 @@ def __str__(self):
         return self.value
 
 
+@dataclass(frozen=True)
+class Referee:
+    referee_id: str
+    name: str = None
+    first_name: str = None
+    last_name: str = None
+    role: Optional[RefereeType] = None
+
+    @property
+    def full_name(self):
+        if self.name:
+            return self.name
+        if self.first_name or self.last_name:
+            return f"{self.first_name} {self.last_name}"
+        if self.role:
+            return f"{self.role}_{self.referee_id}"
```

Comment:
For players, we use a lowercase string if the name is unknown, like `home_{jersey_no}` or `away_{jersey_no}`. For consistency, I would use lowercase for the role of the referee too.